src/Professor.py

```diff
@@ -6,5 +6,4 @@ def __init__(self, name, surname):
         Person.__init__(self, name, surname)
 
     def put_grade(self, grade, student):
-        student.grade.append(grade)
-        student.grade_validation.append(False)
\ No newline at end of file
+        student.grades.append({"value": grade, "valid": False})
```

Comment:
If it is json, so it's even easy to add subject, and probably semester. Like 
student.grades.append({"subject":"SDS", "semester":2, "value": grade, "valid": False})

Reply:
True, I will add this.

Reply:
Updated on 4574cd63aff56409addf6e8cf0ad5565edfd112e.